README.md

```diff
@@ -37,3 +37,24 @@ npm install -g webpack
 npm install
 ```
 
+## API routes
+
+## restaurant-specific routes
+GET /restaurant:id - loads restaurant data using unique restaurant name as parameter
+POST /restaurant - adds new restaurantdata to database, should include data like
+  restaurant name, reservation hours, number of tables, max reservation duration, 
+  max/min number of people for reservation, DOES IT EVEN HAVE A WAITLIST, allowed
+  to make more than one reservation per day
+PUT /restaurant:id - updates restaurant data according to unique restaurant id
+DELETE /restaurant:id - deletes restaurant data according to unique restaurant id
+
+## reservation specific routes
+GET /reservation/:id - get one reservation from the database, requires reservation id
+GET /allReservations/:name - for the restaurant owner side, obtain a list of all
+  the reservations for the restaurant with perhaps an option of selecting by date
+POST /reservation - add reservation to database, should require data like
```

Comment:
Would it make sense to use restaurant name as a param? Maybe we would only want to add a reservation for a specific restaurant? Not sure how your data structure will look like, but this might be something to consider.